src/components/Chat/ChatItem.tsx

```diff
@@ -32,32 +41,36 @@ class ChatItem extends UIComponent<Extendable<ChatItemProps>, any> {
     ...commonPropTypes.createCommon({
       content: 'shorthand',
     }),
+    gutter: customPropTypes.itemShorthand,
+    mine: PropTypes.bool,
   }
 
   static defaultProps = {
     as: 'li',
   }
 
-  renderComponent({
-    ElementType,
-    classes,
-    styles,
-    variables,
-    rest,
-  }: RenderResultConfig<ChatItemProps>) {
-    const { children, content } = this.props
+  renderComponent({ ElementType, classes, rest, styles }: RenderResultConfig<ChatItemProps>) {
+    const { children } = this.props
 
     return (
       <ElementType {...rest} className={classes.root}>
-        {childrenExist(children)
-          ? children
-          : Slot.create(content, {
-              styles: styles.content,
-              variables: variables.content,
-            })}
+        {childrenExist(children) ? children : this.renderChatItem(styles)}
       </ElementType>
     )
   }
+
+  private renderChatItem(styles: ComponentSlotStylesPrepared) {
+    const { content, gutter, mine } = this.props
+    const gutterElement = Slot.create(gutter, { defaultProps: { styles: styles.gutter } })
+
+    return (
+      <>
+        {!mine && gutterElement}
+        {Slot.create(content, { defaultProps: { styles: styles.content } })}
+        {mine && gutterElement}
+      </>
```

Comment:
Should we provide other property for position the gutter like `gutterPosition=start|end` Not sure whether the mine prop is the best option for this.

Reply:
let's discuss this during sync, will keep conversation open

Reply:
still need to discuss this on Friday, @mnajdova can you please follow up on your proposal in the sync on Friday? I'll be on vacation

Reply:
implemented `gutterPosition=start|end`